Mathlib/RingTheory/WittVector/Defs.lean

```diff
@@ -52,7 +52,6 @@ structure WittVector (p : ℕ) (R : Type*) where mk' ::
   -/
   coeff : ℕ → R
 
--- Porting note: added to make the `p` argument explicit
 /-- Construct a Witt vector `mk p x : 𝕎 R` from a sequence `x` of elements of `R`. -/
```

Comment:
Could you explain the exact relation with `mk'` in the docstring
```suggestion
/-- Construct a Witt vector `mk p x : 𝕎 R` from a sequence `x` of elements of `R`.

This is preferred over `WittVector.mk'` because it has `p` explicit.
-/
```
(Is it preferred, or simply an alternative?)